### PR TITLE
ui

### PR DESCRIPTION
**Pull request by yuvraajnarula**

### CODE REVIEW

File: lib/screens/login.dart [hunk 1]
```diff
@@ -20,27 +21,38 @@ class _LoginPageState extends State<LoginPage> {
...
               child: Column(
                 mainAxisAlignment: MainAxisAlignment.center,
                 children: [
-                  Text(
-                    'Login',
-                    style: TextStyle(
-                      fontSize: 36.0,
-                    ),
-                  ),
+                 // Text(
```
Comment (ion05) [inline lib/screens/login.dart:43]: remove karde if not needed

File: lib/screens/registration.dart [hunk 2]
```diff
@@ -21,29 +23,44 @@ class _RegistrationState extends State<Registration> {
...
-                        decoration: InputDecoration(hintText: 'Fullname'),
+                        decoration: InputDecoration(
+                          hintText: 'Fullname',
+                        ),
                       ),
                       TextFormField(
                         decoration: InputDecoration(hintText: 'Username'),
                       ),
                       TextFormField(
-                        decoration: InputDecoration(hintText: 'email'),
```
Comment (ion05) [inline lib/screens/registration.dart:46]: kuch aur toh daal input decor

**Review by ion05 — CHANGES_REQUESTED** [review]

**Comment by ion05** [comment]
Put antonio file in fonts folder inside assets

**Comment by yuvraajnarula** [comment]
flutter_01

**Comment by yuvraajnarula** [comment]
flutter_02

**Comment by ion05** [comment]
1) Gender field ki decoration change karde a bit, looks weird.
2)hint text mein jo written like age limit nd all - make it error text or some other text, not inside the field or tooltip

**Comment by yuvraajnarula** [comment]
flutter_03

File: lib/screens/login.dart [hunk 3]
```diff
@@ -52,6 +52,12 @@ class _LoginPageState extends State<LoginPage> {
                   TextFormField(
                     decoration: InputDecoration(
                       hintText: 'Email',
+                      suffixIcon: IconButton(
```
Comment (ion05) [inline lib/screens/login.dart:55]: make it prefix icon and use Icon

**Review by ion05 — CHANGES_REQUESTED** [review]

**Comment by yuvraajnarula** [comment]
flutter_02

**Comment by yuvraajnarula** [comment]
flutter_01

File: lib/screens/login.dart (hunk 3 above)
Comment (yuvraajnarula) [reply lib/screens/login.dart:55]: ok

**Review by ion05 — APPROVED** [review]

**Pull request merged**